sympy/core/evalf.py

```diff
@@ -208,11 +208,21 @@ def complex_accuracy(result):
 
 
 def get_abs(expr, prec, options):
+    from sympy.functions.elementary.complexes import Abs
```

Comment:
`evalf` is a core module that acts as an interface to mpmath. It should not import from non-core modules of SymPy.